File: SecondYear/Anantj2499_AnantJaiswal_2226cseml1088_2/Week_2/CheckPalindrome.java
public class CheckPalindrome {
        public static boolean palindromeNumber(int n){
            // Write your code here.
            int temp = n;
            int sum =0;
            while(temp>0){
                sum = sum*10+temp%10;
                temp/=10;
            }
            return sum == n;
        }
}

File: SecondYear/adityshrma08_AdityaSharma_2226csit1014_2/Week_2/Check Palindrome.cpp
bool palindrome(int n)
{
    // Write your code here
    int rev=0,temp,digit;
    temp=n;
    while(n>0){
        digit = n%10;
        rev = rev*10 + digit;
        n = n/10;
    }
    if(rev==temp){
        return true;
    }
    else
    return false;
}